mypy/test/config.py

```diff
@@ -1,12 +1,13 @@
 from __future__ import annotations
 
 import os.path
+from pathlib import PurePath
 
 provided_prefix = os.getenv("MYPY_TEST_PREFIX", None)
 if provided_prefix:
     PREFIX = provided_prefix
 else:
-    this_file_dir = os.path.dirname(os.path.realpath(__file__))
+    this_file_dir = os.path.dirname(PurePath(__file__))
     PREFIX = os.path.dirname(os.path.dirname(this_file_dir))
```

Comment:
To be clear, I was thinking that we can replace `os.path.dirname` with the relevant pathlib operation (`this_file_dir = (Path(__file__) / '..').resolve()`?)